src/app/components/universal/newSafetyDocumentUpdate/newSafetyDocumentUpdate.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+
+@Component({
+  selector: 'app-newSafetyDocumentUpdate',
+  templateUrl: './newSafetyDocumentUpdate.component.html',
+  styleUrls: ['./newSafetyDocumentUpdate.component.css']
+})
+export class NewSafetyDocumentUpdateComponent implements OnInit{
+
+  safetyDocumentUpdated: boolean;
+
+  @Input() heading: string;
+  @Input() body: string;
+  @Input() alertType: string; // Should be either warning, ??? or ???
+  @Input() addLink: boolean = false; // When no value is given, default to false
+  public alertIconStyle: {[key: string]: string};
+
+  checkSafetyDocumentUpdated(event: boolean) {
```

Comment:
Move to under ngoninit

Add access modifier and return type

Add jsdoc - with why method is needed and what used for.